examples/typescript/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react"
```

Comment:
Any reason for this change? Shall we maybe switch to https://www.npmjs.com/package/@tsconfig/create-react-app for even less code?